modules/claims_api/config/schemas/526.json

```diff
@@ -1091,6 +1091,8 @@
               "name": {
```

Comment:
I'm confused on this because didn't Michael Harlow say that it was the `treatments[].center.country` that had the 1-50 characters requirement? ([REF](https://dsva.slack.com/archives/C8R3JS8BU/p1650304654062279?thread_ts=1650304364.435459&cid=C8R3JS8BU)]

Reply:
ope! you're totally right. Remedying that in my next commit